diff -crN ftb-0.6.2/src/agent/ftb_agent.c ftb-0.6.2.p/src/agent/ftb_agent.c
*** ftb-0.6.2/src/agent/ftb_agent.c	Sat Aug 14 07:58:31 2010
--- ftb-0.6.2.p/src/agent/ftb_agent.c	Fri Apr 26 17:52:04 2013
***************
*** 143,156 ****
      pthread_mutex_init(&lock, NULL);
      done = 0;
  
! 
  #ifndef FTB_CRAY
      int pid;
      pid = fork();
      if (pid != 0)
          return 0;
  #endif
! 
  
  #ifdef FTB_DEBUG
      if (argc >= 2 && strcmp("ION_AGENT", argv[1]) == 0) {
--- 143,156 ----
      pthread_mutex_init(&lock, NULL);
      done = 0;
  
! /* FOR libmicp.a
  #ifndef FTB_CRAY
      int pid;
      pid = fork();
      if (pid != 0)
          return 0;
  #endif
! */
  
  #ifdef FTB_DEBUG
      if (argc >= 2 && strcmp("ION_AGENT", argv[1]) == 0) {
